javagen/src/main/java/com/azure/autorest/model/projectmodel/Project.java

```diff
@@ -50,7 +50,8 @@ public class Project {
 
     public static class PackageVersions {
         private String azureClientSdkParentVersion = "1.7.0";
-        private String azureCoreVersion = "1.29.1";
+        private String azureJsonVersion = "1.0.0-beta.1";
+        private String azureCoreVersion = "1.30.0-beta.1";
```

Comment:
Before releasing autorest, we should update to the GA version of core. Also, are we planning to GA azure-json soon?

Reply:
The beta dependency on Core has been removed since all useful code was moved into azure-json, first release of azure-json is still TBD but there is a private build available to Autorest